web/src/views/Dashboards/Dashboards.vue

```diff
@@ -286,7 +299,7 @@ export default defineComponent({
               JSON.stringify(toRaw(store.state.allDashboardList))
             );
             const newDashboardList = dashboardList.filter(
-              (dashboard) => dashboard.name != dashboardId
+              (dashboard) => dashboard.dashboardId != dashboardId
```

Comment:
@vvv This has been already fixed here. Pending merge. (https://github.com/zinclabs/zincobserve/issues/540)